lesson28/src/register.html

```diff
@@ -14,17 +14,17 @@
         <h1>会員登録</h1>
         <div class="mb-6">
           <label for="user_name">お名前<span class="required text-red-500 mx-1" aria-hidden="true">*</span></label>
-          <input type="text" name="user_name" id="user_name" autocomplete="username" placeholder="山田　太郎" required>
+          <input type="text" name="user_name" id="user_name" data-id= "user_name" autocomplete="username" placeholder="山田　太郎" required>
         </div>
         <div class="mb-6">
           <label for="email">メールアドレス<span class="required text-red-500 mx-1" aria-hidden="true">*</span></label>
-          <input type="email" name="email" id="email" autocomplete="username" placeholder="sample@sample.com" required>
+          <input type="email" name="email" id="email" data-id="email" autocomplete="username" placeholder="sample@sample.com" required>
         </div>
         <div class="mb-6 relative">
           <label for="new-password">パスワード<span class="required text-red-500 mx-1" aria-hidden="true">*</span></label>
           <p id="password-constraints" class="text-[13px] text-gray-500 mb-2">8文字以上の大小英数字を含めてください</p>
           <button id="js-toggle-password" type="button" class="absolute cursor-pointer h-[42px] py-1 px-2 top-[60px] right-0 text-[16px] rounded-sm text-blue-500 font-bold" aria-label="パスワードを表示">表示</button>
-          <input type="password" name="new-password" id="new-password" autocomplete="new-password" placeholder="********" aria-describedby="password-constraints" required>
+          <input type="password" name="new-password" id="new-password" data-id="new-password" autocomplete="new-password" placeholder="********" aria-describedby="password-constraints" required>
```

Comment:
Q. 
I felt that "password" was fine for the data-id here, but is it necessary to add new-?

Reply:
@haru-programming 
Thank you for your comment!
I don't need new because there is no other password element to distinguish between the two!
I fixed.